ecs-cli/modules/cli/local/create_app.go

```diff
@@ -18,12 +18,39 @@ package local
 import (
 	"fmt"
 
+	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/commands/flags"
 	"github.com/urfave/cli"
 )
 
+const (
+	// taskDefinitionLabelType represents the type of option used to
+	// transform a task definition to a compose file e.g. remoteFile, localFile.
+	// taskDefinitionLabelValue represents the value of the option
+	// e.g. file path, arn, family.
+	taskDefinitionLabelType  = "ecsLocalTaskDefType"
+	taskDefinitionLabelValue = "ecsLocalTaskDefVal"
+)
+
+const (
+	localTaskDefType  = "localFile"
+	remoteTaskDefType = "remoteFile"
+)
```

Comment:
I'm also a little confused why these four constants are defined in `create_app.go` right now, as I don't see them being used in the create function. Perhaps it would make sense to move these constants to a different file still under the `local` package namespace?

Reply:
I think the idea is that eventually `Create()` will eventually add these as Docker labels to the containers so that we know if the containers were started with a `localFile` or `remoteFile`.